flink-runtime/src/test/java/org/apache/flink/runtime/io/network/NettyShuffleEnvironmentBuilder.java

```diff
@@ -180,7 +187,8 @@ public NettyShuffleEnvironment build() {
                         compressionCodec,
                         maxBuffersPerChannel,
                         sortShuffleMinBuffers,
-                        sortShuffleMinParallelism),
+                        sortShuffleMinParallelism,
+                        batchShuffleReadBuffers),
```

Comment:
I guess it should be `batchShuffleReadBytes` and it should be placed in ahead of the param `sortShuffleMinBuffers`.